input/resources/valueset-rsg-type.xml

```diff
@@ -12,11 +12,11 @@
         <system value="urn:ietf:rfc:3986"/>
         <value value="urn:oid:2.16.840.1.113883.2.3.4.2.4.31"/>
     </identifier>
-    <name value="CommonAURecordedSexOrGenderType" />
-    <title value="Common AU Recorded Sex or Gender Type" />
+    <name value="AURecordedSexOrGenderType" />
+    <title value="AU Recorded Sex or Gender Type" />
```

Comment:
These changes are not related to FHIR-46107, please revert.

Reply:
Reverted change

Reply:
Change reverted